.github/workflows/02-e2e-screen-reader.yml

```diff
@@ -39,12 +39,24 @@ jobs:
            echo "BROWSER=webkit" >> "$GITHUB_ENV"
           fi
 
+
+      - uses: actions/cache@v4
+        id: playwright-cache
+        with:
+          path: '~/.cache/ms-playwright'
+          key: '${{ runner.os }}-playwright-1.43.1'
+          restore-keys: |
+            ${{ runner.os }}-playwright-
+
+      - name: Install Playwright's dependencies
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+        run: npx playwright install --with-deps ${{ env.BROWSER }}
+
```

Comment:
this looks like an optimization independently from screen-reader test integration to me, isn' it ?